crates/typst/src/model/bibliography.rs

```diff
@@ -44,37 +44,31 @@ use crate::text::{
 
-/// A bibliography / reference listing.
+/// 参考文献 / 引用文献リスト。
 ///
-/// You can create a new bibliography by calling this function with a path
-/// to a bibliography file in either one of two formats:
+/// 次の2つの形式のどちらかの参考文献ファイルへのパスを指定してこの関数を呼び出すと、新しい引用文献リストを作成できます。
 ///
-/// - A Hayagriva `.yml` file. Hayagriva is a new bibliography file format
-///   designed for use with Typst. Visit its
-///   [documentation](https://github.com/typst/hayagriva/blob/main/docs/file-format.md)
-///   for more details.
-/// - A BibLaTeX `.bib` file.
+/// - Hayagriva `.yml` ファイル。
+///   HayagrivaはTypstで使用するためにデザインされた新しい書誌ファイルフォーマットです。
+///   詳しくは[ドキュメント](https://github.com/typst/hayagriva/blob/main/docs/file-format.md)をご覧ください。
+/// - BibLaTeX `.bib` ファイル。
 ///
-/// As soon as you add a bibliography somewhere in your document, you can start
-/// citing things with reference syntax (`[@key]`) or explicit calls to the
-/// [citation]($cite) function (`[#cite(<key>)]`). The bibliography will only
-/// show entries for works that were referenced in the document.
+/// 文書内に参考文献を追加すると、参照構文（`[@key]`）や引用関数の明示的な呼び出し（`[#cite(<key>)]`）を使って[引用]($cite)を始めることができます。
+/// 参考文献リストには、文書内で参照された作品の文献だけが表示されます。
 ///
-/// # Styles
-/// Typst offers a wide selection of built-in
-/// [citation and bibliography styles]($bibliography.style). Beyond those, you
-/// can add and use custom [CSL](https://citationstyles.org/) (Citation Style
-/// Language) files. Wondering which style to use? Here are some good defaults
-/// based on what discipline you're working in:
+/// # スタイル
+/// Typstは、内蔵の[引用と文献スタイル]($bibliography.style)を幅広く取り揃えています。
+/// さらに、独自の[CSL](https://citationstyles.org/)（Citation Style Language）ファイルを追加して使用することもできます。
+/// どのスタイルを使えばいいか迷う方のために、分野ごとによく使われるスタイルを以下の表にまとめています。
 ///
-/// | Fields          | Typical Styles                                         |
+/// | 分野     | Typical Styles                                                |
 /// |-----------------|--------------------------------------------------------|
-/// | Engineering, IT | `{"ieee"}`                                             |
-/// | Psychology, Life Sciences | `{"apa"}`                                    |
-/// | Social sciences | `{"chicago-author-date"}`                              |
-/// | Humanities      | `{"mla"}`, `{"chicago-notes"}`, `{"harvard-cite-them-right"}` |
-/// | Economics       | `{"harvard-cite-them-right"}`                          |
-/// | Physics         | `{"american-physics-society"}`                         |
+/// | 工学、IT | `{"ieee"}`                                                    |
+/// | 心理学、ライフサイエンス | `{"apa"}`                                       |
+/// | 社会科学 | `{"chicago-author-date"}`                                     |
+/// | 人文学   | `{"mla"}`, `{"chicago-notes"}`, `{"harvard-cite-them-right"}` |
+/// | 経済学   | `{"harvard-cite-them-right"}`                                 |
+/// | 物理学   | `{"american-physics-society"}`                                |
 ///
 /// # Example
```

Comment:
ここの見出しも訳した方が良いと思います？

```suggestion
/// # 例
```

Reply:
反映させていただきました。